usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -9042,10 +9041,8 @@ static const CK_MECHANISM_TYPE ep11_supported_mech_list[] = {
     CKM_DES3_KEY_GEN,
     CKM_DH_PKCS_DERIVE,
     CKM_DH_PKCS_KEY_PAIR_GEN,
-    CKM_DH_PKCS_PARAMETER_GEN,
```

Comment:
Just make sure the book correctly documents the same.

Reply:
Hm, right. Unfortunately, the book includes these mechs... Will make sure the book gets updated.